site/en/tutorials/text/classify_text_with_bert.ipynb

```diff
@@ -952,7 +952,7 @@
         "id": "3cOmih754Y_M"
       },
       "source": [
-        "If you want to use your model on [TF Serving](https://www.tensorflow.org/tfx/guide/serving), remember that it will call your SavedModel through one of its named signatures. In Python, you can test them as follows:"
+        "If you want to use your model on [TF Serving](https://www.tensorflow.org/tfx/guide/serving), remember that it will call your SavedModel through one of it's named signatures. In Python, you can test them as follows:"
```

Comment:
Revert this one. Review your "it's vs its". 

 "it's" is "it is" or "it has".
 
  "its" is like "his", "hers", or "theirs".

Reply:
Yes, it has lost _its_ meaning 😐